### DISCUSSION TITLE
Copyright to be included for NuGet packate

### DISCUSSION THREAD

**Discussion by manjunthcm** [dlemstra/Magick.NET#630] [General] [answered]
hi 
Sorry if this is naive question.
The license link in NuGet package manager, while installing Magick.Net points to https://licenses.nuget.org/Apache-2.0 which states that we include the boilerplate notice given at the end in that link be included to apply the Apache License to our work.

But the Copyright.txt at **..\packages\Magick.NET-Q16-AnyCPU.7.16.1**, gives copyright in different format and different license and points to https://www.imagemagick.org/script/license.php 

please help me in understanding which one needs to be used ? [ though it is basic, i thought of getting this clarified ]

any help is highly appreciated.

**Comment by dlemstra** [comment, marked as answer]
I am not a lawyer so I am not 100% sure what is correct. Thank you for the extra effort to confirm to this. I would advise you to copy the Copyright.txt file and add your own copyright at the top of the file. I have also seen others use the name `THIRD-PARTY-LICENSES.txt` and include that in their project. Either is fine with me.

**Comment by manjunthcm** [comment]
Thanks for the confirmation
